partcad/src/partcad/part_factory_obj.py

```
@@ -0,0 +1,152 @@
+                return True
+
+    def _load_obj_directly(self):
+        """
+        Load an OBJ file directly in the main thread.
+        """
+        time.sleep(0.0001)  # Brief pause for thread synchronization
+        try:
+            vertices = []
+            faces = []
+
+            with open(self.path, 'r') as file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)